src/db/migrations/migration1.js

```diff
@@ -0,0 +1,31 @@
+const logger = require('../../utils/logger');
+const DBHelper = require('../db-helper');
+const { TX_STATUS } = require('../../constants');
+
+async function migration1() {
+  try {
+    const withdraws = await DBHelper.findWithdraw({ txStatus: TX_STATUS.SUCCESS });
+    const withdrawnLists = new Map();
+    for (let i = 0; i < withdraws.length; i++) {
+      const withdraw = withdraws[i];
+      let withdrawnList = [];
+      if (withdrawnLists.has(withdraw.eventAddress)) {
+        withdrawnList = withdrawnLists.get(withdraw.eventAddress);
+      }
+      withdrawnList.push(withdraw.winnerAddress);
+      withdrawnLists.set(withdraw.eventAddress, withdrawnList);
+    }
+
+    withdrawnLists.forEach((value, key) => {
```

Comment:
Need to create a list of promises, add all the update promises, then call `await Promise.all`. This current logic wont wait for all the db updates to be done before continuing.